### PR TITLE
Remove bad code

### PR DESCRIPTION
**Pull request by SlavaRa**
@Neverbirth please review this PR and merge him :)

### CODE REVIEW

File: External/Plugins/StartPage/PluginMain.cs [hunk 1]
```diff
@@ -128,15 +128,12 @@ public void HandleEvent(Object sender, NotifyEvent e, HandlingPriority priority)
                             if (de.Data != null & this.startPage != null)
```
Comment (Neverbirth) [inline External/Plugins/StartPage/PluginMain.cs]: de.Data != null & this.startPage != null this should be && heh.

Comment (SlavaRa) [inline External/Plugins/StartPage/PluginMain.cs]: Fixed

**Comment by Neverbirth** [comment]
Please, revert the change to PluginCore/BBCode/BottomUpParser.cs and all the ones inside PluginCore/DockPanelSuite/*.

Since they don't really add improvements it's better to avoid those ones for possible future improvements (like updating DockPanelSuite), and because BBCode from my side is removed and hopefully I can see it replaced soon...

**Comment by SlavaRa** [comment]
One moment

**Comment by SlavaRa** [comment]
Done

**Pull request merged**